src/extension.ts

```diff
@@ -255,6 +255,28 @@ export class Extension implements vscode.Disposable {
     return this.enabled = enabled
   }
 
+  /**
+   * Make all selections in the editor non-empty by selecting at least one character.
+   */
+  normalizeSelections(editor: vscode.TextEditor) {
```

Comment:
Since all callers will be in the same class, this can be made `private`.

Reply:
Done.